client/lib/plans/constants.js

```diff
@@ -82,6 +83,7 @@ export const plansList = {
 		getDescription: () => i18n.translate( 'Get a free website and be on your way to publishing your ' +
 			'first post in less than five minutes.' ),
 		getFeatures: () => [ // pay attention to ordering, it is used on /plan page
+			FEATURE_JETPACK_ESSENTIAL,
```

Comment:
I'd move this below `FEATURE_WP_SUBDOMAIN`, as it coincides with the `FEATURE_CUSTOM_DOMAIN` in the other plans. Doesn't really matter for now because we aren't displaying the free plan at the moment.